pkg/scheduler/service/bookkeeper.go

```diff
@@ -131,6 +139,16 @@ func (bk *bookkeeper) finishReconciliation(reconResult *ReconciliationResult) bo
 	recon := reconResult.Reconciliation()
 	newClusterStatus := reconResult.GetResult()
 
+	if newClusterStatus == model.ClusterStatusReconcileError {
+		errCnt, err := bk.transition.inventory.CountRetries(reconResult.reconEntity.RuntimeID, reconResult.reconEntity.ClusterConfig)
+		if err != nil {
+			bk.logger.Errorf("failed to count error for runtime %s with error: %s", reconResult.reconEntity.RuntimeID, err)
+		}
+		if errCnt < bk.config.MaxRetries {
+			newClusterStatus = model.ClusterStatusReconcileErrorRetryable
+		}
```

Comment:
Please add also an INFO log message that a cluster with runtimeID 'X' and clusterConfig 'Y' had a failing reconcilign but was retried because it had just 'Z' retries.

Reply:
done